src/AssetManager.php

```diff
@@ -834,16 +834,22 @@ private function isAllowedBundleDependencies(string $name, AssetBundle $bundle):
         return false;
     }
 
-    private function mergeWithReverseOrder(array $a, array $b): array
+    /**
+     * @throws InvalidConfigException
+     */
+    private function mergeOptionsWithArray(array $options, array $array): array
     {
-        foreach ($a as $key => $value) {
+        foreach ($options as $key => $value) {
             if (is_int($key)) {
-                $b[] = $value;
-            } elseif (!array_key_exists($key, $b)) {
-                $b[$key] = $value;
+                throw new InvalidConfigException(
+                    'JavaScript or CSS options should be list of key/value pairs with string ketys. Got integer key.'
```

Comment:
'JavaScript or CSS options should be list of key/value pairs with string keys. Got integer key.'

Reply:
what is the question?

Reply:
just correct keys 
It's my first time

Reply:
Thank you. 
I didn't notice it at first.